content/get-started/index.md

```diff
@@ -0,0 +1,78 @@
+| Your first Galaxy Analysis | [<i class="fab fa-slideshare" aria-hidden="true"></i>](bit.ly/smorg-2021-intro){:target="_blank"}  |   |[<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/1UPCrQxGwcU){:target="_blank"} |   |
+| Galaxy Tour  |   |   |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.galaxy_ui){:target="_blank"} |
+| Galaxy Histories |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html){:target="_blank"}  |   | [<i class="fas fa-mouse" aria-hidden="true"></i>](https://usegalaxy.org/tours/core.history){:target="_blank"} |
+| How to reproduce published Galaxy Analysis |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-reproduce/tutorial.html){:target="_blank"} |   |   |
+|Introduction to the GTN |[<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/slides/introduction.html#1
+){:target="_blank"} |   | [<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=lDqWxzWNk1k){:target="_blank"} |   |
+| How to contribute using Gitpod |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/gitpod/tutorial.html){:target="_blank"}  | [<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/_-wnsPttBCo){:target="_blank"}  |   |
+|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |
+
+### Useful links
+- [FAQs quickstart](https://training.galaxyproject.org/training-material/faq)
+  
+  Feel like you might have a question that many others have had as well? Check our Frequently Asked Questions quickstart!
+
+- [Public Galaxy Servers](https://galaxyproject.org/use/)
+  
+  List of all Public Galaxy Servers available to use or deploy your own version.
+
+- [Galaxy Training Network](https://training.galaxyproject.org/)
+  
+  Collection of tutorials from a variety of topics() developed and maintained by the worldwide Galaxy community.
+
+- [Galaxy Help](https://help.galaxyproject.org/)
+  
+  Have a bug or an error you haven't been able to solve? Do you have a question that you haven't found in already answered in the FAQs? Reach out to the Galaxy Help Community.
```

Comment:
I would also probably put this after the galaxy server and before tutorial (not sure). Because when you start a tutorial, you want to know where you can ask for help if you get stuck.